Integrations/integration-IntSights.yml

```diff
@@ -123,6 +123,23 @@ script:
         return alerts
 
 
+    def extract_mail(replies):
+        mails = []
+        for reply in replies:
+            mails.append(reply.get('Email'))
+
+        return '\n'.join(mails)
+
+
+    def extract_remediation(remidiations):
+        remdies = []
+        string_format = "{0} - Status: {1}"
+        for remdy in remidiations:
+            remdies.append(string_format.format(remdy.get('Value'), remdy.get('Status')))
```

Comment:
If `Value` or `Status` keys will not be in `remdy`, the output would be `None - Status: None`. Can we handle that better?

Reply:
👍